DAQ/src/CaloHitsFromFragments_module.cc

```diff
@@ -68,8 +68,17 @@ class art::CaloHitsFromFragments : public EDProducer {
         fhicl::Name("nSigmaNoise"),
         fhicl::Comment("Maxnumber of sigma Noise to combine digi") };
     fhicl::Atom<float> hitEDepMax{
-        fhicl::Name("hitEDepMax"),
+        fhicl::Name("hitEDepMin"),
```

Comment:
There is now a duplicate definition for the fhicl::Name("hitEDepMin"). I presume this is a typo?

Reply:
fixed. Thanks for noticing it.